### PR TITLE
admin2: Fix Anonymous Admin not working

### PR DESCRIPTION
**Pull request by xLuxy**
Add missing line change introducted in one of my recently commited (https://github.com/multitheftauto/mtasa-resources/commit/1b99a05b948abd74df5cea97b7f5b71ed0b22be9) - it was removed somehow

// cc @qaisjp

### CODE REVIEW

**Comment by qaisjp** [comment]
I made this change. What's wrong with it?

**Comment by xLuxy** [comment]
> I made this change. What's wrong with it?

In the commit you've made there's 1 change missing which hides the admins nickname. In current master, Anonymous mode has no effect as the name will be displayed regardless of the state.

**Comment by qaisjp** [comment]
Please can you revert the `hex or ''` change

**Comment by xLuxy** [comment]
done

File: [admin]/admin2/server/admin_server.lua [hunk 1]
```diff
@@ -103,7 +103,7 @@ function aAction(type, action, admin, player, data, more)
                 hex = ""
             end
             string = tostring(string)
-            string = string.gsub(string, "$admin", getPlayerName(admin) .. hex)
+            string = string.gsub(string, "$admin", (not isAnonAdmin(admin) and getPlayerName(admin) .. hex or "Admin"))
```
Comment (qaisjp) [inline [admin]/admin2/server/admin_server.lua:106]:
```suggestion
            string = string.gsub(string, "$admin", isAnonAdmin(admin) and "Admin" or (getPlayerName(admin) .. hex))
```

Comment (xLuxy) [reply [admin]/admin2/server/admin_server.lua]: done

**Pull request merged**